### PR TITLE
Fix version check workflow

### PR DESCRIPTION
**Pull request by dinakar29**
Encountered a bug in the check version workflow. Fix is here.

### CODE REVIEW

**Review by nikolas-burkoff — APPROVED** [review]

File: DESCRIPTION [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Package: staged.dependencies
 Type: Package
 Title: Install R packages from Particular Git Branches
-Version: 0.2.4
+Version: 0.2.5
```
Comment (nikolas-burkoff) [inline DESCRIPTION:4]: actually should we set this to 0.2.4.9001 so that only when we "release" do we get 0.2.5?

Comment (nikolas-burkoff) [reply DESCRIPTION:4]: Ah I think that was in preparation for next release (there wasn't a version bump action etc. here)

**Comment by dinakar29** [comment]
@nikolas-burkoff - do you mind if I upgrade dependencies in `renv.lock`?

**Comment by nikolas-burkoff** [comment]
> @nikolas-burkoff - do you mind if I upgrade dependencies in renv.lock?

Nope that's fine with me (assuming tests pass of course!)

**Comment by nikolas-burkoff** [comment]
> @nikolas-burkoff - do you mind if I upgrade dependencies in renv.lock?

I've done this here https://github.com/openpharma/staged.dependencies/pull/145

**Comment by nikolas-burkoff** [comment]
So in TL;DR form - how does this impact me? ;)

**Comment by dinakar29** [comment]
> So in TL;DR form - how does this impact me? ;)

You can now run workflows on demand, any time manually. That's the big one.

Also, there's proper support for renv now across workflows, so this PR was sort of a test bed to make sure our common workflows are compatible with repos that have renv.lock files.

**Comment by nikolas-burkoff** [comment]
Cool thanks!

**Pull request merged**